pkgs/tools/admin/google-cloud-sdk/default.nix

```diff
@@ -68,7 +67,7 @@ in stdenv.mkDerivation rec {
     disable_update_check = true" >> $out/google-cloud-sdk/properties
 
     # setup bash completion
-    mkdir -p "$out/etc/bash_completion.d/"
+    mkdir -p $out/etc/bash_completion.d/
```

Comment:
that change isn't necessary

Reply:
yeah, that was just to remove the superfluous double quotes and match style with the rest - we don't have spaces in `$out` etc.

In fact, I should change the line below that one too :-)

Reply:
done. :-)